src/pages/List/ListData.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+const ListData = ({ data }) => {
+  const { title, price, gram, img_url, antibiotic } = data;
+  return (
+    <li className="list_data_container">
+      <div className="list_img">
+        {antibiotic ? <span>무항생제</span> : null}
```

Comment:
falsy한 경우의 값이 null이라면 논리곱 단축평가를 활용하시면 될 것 같습니다.
```suggestion
        {antibiotic && <span>무항생제</span>}
```